src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -247,7 +247,7 @@ In order to debug this project, add an executable project to this solution which
     <value>Expected to find migrated cpsroj in {0}, but did not find any.</value>
   </data>
   <data name="XprojMigrationFailed" xml:space="preserve">
-    <value>Failed to migrate XProj project {0}. 'dotnet migrate --skip-backup -s -p "{1}" -x "{2}"' exited with error code {3}.</value>
+    <value>Failed to migrate XProj project {0}. 'dotnet migrate --skip-backup -s -x "{1}" "{2}" -r "{3}" --format-report-file-json' exited with error code {4}.</value>
```

Comment:
Careful changing placeholder count, position, meaning. Removing them is very bad as it will cause the builds between loc handoff/handback to crash in this case (format exception). 

This won't crash, but a localized build that doesn't yet have the corresponding translation update would incorrectly display the argument to -r as the error code and drop the actual error code.

I don't think the migration command should be in a resource. It is not localizable and whenever it changes, we're going to be in the same complex situation.

I suggest:

1. Choose a new name
2. Change the value to `Failed to migrate XProj project {0}. '{1}' exited with error code {2}.